index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+    <script src="app.js"></script>
+  </head>
+  <body>
+
+    <header class="header-images">
+      <div class="background2Box">
+      <img class="logoHeader"src="./images/adoptadog-logo.svg" alt="Company Logo" id="adoptadog-logo">
+      <h2 class="headertext"> Seattle's premiere dog adoption agency </h2>
+        </div>
+      <a class="checkout" href="./checkout.html">Checkout</a>
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="./dogs.html">Dogs</a>
+        </li>
+        <li>
+          <a href="./blog.html">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main id="wrapper">
+      <section class="flex-container">
+        <h2 class="hide">Section Header</h2>
+          <article class="div1">
+            <h2>Adopt A Dog</h2>
+            <img src="./images/home-intro.jpg" alt="A man with a dog">
+
+            <h2 id="a1">Adopt A Dog</h2>
+
+            <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate
+</p>
+
+          </article>
+
+          <article class="div2">
```

Comment:
I would argue these are not articles. a div would be fine.

Also use a semantic name for this class. div2 is not very descriptive. Something like dog-tile would be better.

Reply:
thanks for suggestion. I will update it